admin.php

```diff
@@ -25,10 +25,11 @@
 }
 
 $data = App::getFeed();
-$isNewVersionAvailable = isset($data['version']) && $data['version'] != '' && $data['version'] !== Array();
+$isNewVersionAvailable = isset($data['version']) && !empty($data['version']) && $data['version'] !== Array();
```

Comment:
`$data['version'] !== Array()` is not required anymore, since that would be matched by `empty($data['version']`